src/Tools/Service/Alerting/Adapter/Sns.php

```diff
@@ -33,9 +33,20 @@ class Sns implements AdapterInterface
      */
     protected function getClient()
     {
+        $config = $this->getServiceLocator()->get('config');
```

Comment:
The config is not typed, controlled, this should be an Object to ensure the content or have strict checker on it.
In most of our case we must setup the credential as Profile, key/secret is not recommended.